### PR TITLE
zh-cn local for Download page(index/current)

### PR DESCRIPTION
**Pull request by undownding**
And, some files in `layouts/partials/` need to i18n.

See `layouts/partials/secondary-download-matrix.hbs` in this PR.

### CODE REVIEW

**Comment by undownding** [comment]
`release.md` will translate later...

**Comment by ghost** [comment]
ping @nodejs/nodejs-cn

**Comment by detailyang** [comment]
:beers: LGTM for Chinese translation

**Comment by ghost** [comment]
LGTM from me as well then!

File: locale/zh-cn/download/releases.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: download-releases.hbs
+title: Previous Releases
+modules: "<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. It used to be stored as hex value in earlier versions, but is now represented as an integer."
+---
+
+### io.js &amp; Node.js
+Releases 1.x through 3.x were called "io.js" as they were part of the io.js fork. As of Node.js 4.0.0 the former release lines of io.js converged with Node.js 0.12.x into unified Node.js releases.
```
Comment (popomore) [inline locale/zh-cn/download/releases.md:8]: Why not translate this page?

Comment (lpinca) [inline locale/zh-cn/download/releases.md:8]: From @undownding 

> release.md will translate later...

Premature merge, I guess.

Comment (ghost) [inline locale/zh-cn/download/releases.md:8]: nah, the pr title specifically says `(index/current)` so i assumed the other ones were going to be translated another time

Comment (popomore) [inline locale/zh-cn/download/releases.md:8]: I missed this comment...

Comment (undownding) [inline locale/zh-cn/download/releases.md:8]: I forgot translated it before I `git add`.... will fix it tomorrow.Sorry about that.

**Comment by stevemao** [comment]
I'd merge it a bit late (until 2-3 i18n +1s). But not a big deal I guess :)

**Pull request merged**